os/arch/arm/src/amebasmart/amebasmart_mipi.c

```diff
@@ -322,12 +325,42 @@ static int amebasmart_mipi_transfer(FAR struct mipi_dsi_host *dsi_host, FAR cons
 	return OK;
 }
 
+#ifdef CONFIG_PM
+static uint32_t rtk_mipi_suspend(uint32_t expected_idle_time, void *param)
+{
+	(void)expected_idle_time;
+	(void)param;
+
+	/* Check anything is needed? */
+	return 1;
+}
+
+static uint32_t rtk_mipi_resume(uint32_t expected_idle_time, void *param)
+{
+	(void)expected_idle_time;
+	(void)param;
+
+	FAR struct amebasmart_mipi_dsi_host_s *priv = &g_dsi_host;
+	/* For MIPI */
+	amebasmart_mipi_init_helper(priv);
+	/* For LCDC */
+	rtl8730e_lcdc_pm();
+
+	return 1;
+}
+#endif
+
 struct mipi_dsi_host *amebasmart_mipi_dsi_host_initialize(struct lcd_data *config)
 {
 	FAR struct amebasmart_mipi_dsi_host_s *priv = NULL;
 	priv = &g_dsi_host;
 	priv->dsi_host.config = *config;
 	amebasmart_mipi_init_helper(priv);
 	mipi_dsi_host_register(&priv->dsi_host);
+#ifdef CONFIG_PM
+	(void)pm_domain_register("LCD");
```

Comment:
hmmm. How about `mipi` instead of `lcd`? Because the LCD driver also needs to check pm.

Reply:
I will change this in this PR: https://github.com/Samsung/TizenRT/pull/6256